src/main/webapp/app/exam/manage/test-runs/test-run-management.component.ts

```diff
@@ -20,6 +20,13 @@ import { faSort, faTimes } from '@fortawesome/free-solid-svg-icons';
     templateUrl: './test-run-management.component.html',
 })
 export class TestRunManagementComponent implements OnInit {
+    private route = inject(ActivatedRoute);
+    private alertService = inject(AlertService);
+    private examManagementService = inject(ExamManagementService);
+    private accountService = inject(AccountService);
+    private sortService = inject(SortService);
+    private modalService = inject(NgbModal);
+
     course: Course;
     exam: Exam;
     isLoading: boolean;
```

Comment:
can you use signals for the variables here?

Reply:
done